lib/io/hardware_gpio.hpp

```diff
@@ -2,31 +2,63 @@
 
 #include "gpio.hpp"
 
+#include <unordered_map>
+
 #include <core/types.hpp>
 
 namespace hyped::io {
 
+// GPIO hardware specified addresses and sizes for read, clear, set, size, etc.
+static constexpr std::uint32_t pin_size  = 0x1000;
```

Comment:
`kPinSize`?

Reply:
Also this feels like quite a high scope to be declaring these? Is there absolutely nothing else in `hyped::io` that could be called `pin_size`?